### ISSUE TITLE
Maybe add a "snow" tag for Surface Problem?

### ISSUE THREAD

**Issue by misaugstad** [ProjectSidewalk/SidewalkWebpage#3386]
##### Brief description of problem/feature
It looks like a bunch of the recent drive through of Teaneck, NJ happened during the winter with a bunch of snow on the sidewalks. I've been suggesting that people use the "can't see the sidewalk" label when snow is covering corner and you don't know whether or not there are curb ramps. But when snow is on the rest of the sidewalk, a maybe a surface problem label should be used?

I'm wondering if we could even find a way to wrap up seasonal issues like snow, fallen leaves, and maybe even pooled water into a single tag?

**Comment by misaugstad** [comment]
Could maybe add this tag to the "Can't See the Sidewalk" label type instead?

**Comment by misaugstad** [comment]
I would love to have people using the "Can't See the Sidewalk" label type when the sidewalk is covered with snow, but I just don't really expect that to happen! I think it makes sense to just add a "snow" tag to the Surface Problem label type, so that users of our API can filter out such labels if they want to.

@nehaa2426 I'll ask you to work on this! You can use PR #3656 as a reference. I found an example image for you:
![Screenshot from 2024-10-09 14-16-00](https://github.com/user-attachments/assets/af1af63c-7461-4193-ac51-69decbf75c0f)

**Comment by nehaa2426** [comment]
@misaugstad would you like a shortcut label for this, and which character

**Comment by misaugstad** [comment]
Yes for shortcut, and do your best to find a character that is still available! Maybe send me the list of available ones if you like? I realize that the list of shortcuts for Surface Problem labels has grown very thin! Note that they also can't overlap with other shortcuts on the UI, not just the tag shortcuts.

**Comment by misaugstad** [comment]
Had another request from a volunteer today to add a 'snow' tag! 

<img width="1323" height="973" alt="Image" src="https://github.com/user-attachments/assets/7cccd9fd-c600-446c-94e7-f987466de6df" />
